astropy/utils/iers/iers.py

```diff
@@ -599,7 +599,7 @@ def _check_interpolate_indices(self, indices_orig, indices_clipped, max_input_mj
 
         # See explanation in _refresh_table_as_needed for these conditions
         auto_max_age = (conf.auto_max_age if conf.auto_max_age is not None
-                        else np.finfo(np.float).max)
```

Comment:
As far as `utils` is concerned, I defer IERS here and metadata below to @taldcroft .

Reply:
This one is OK.